### PR TITLE
Tests for broker broker non existing external topic

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

## Proposed Changes

- two more tests for non-existing external topic

### CODE REVIEW

File: test/rekt/features/broker_deleted_recreated.go [hunk 1]
```diff
@@ -131,6 +131,63 @@ func BrokerAuthSecretDoesNotExist() *feature.Feature {
...
+	brokerName := feature.MakeRandomK8sName("broker")
+	configName := feature.MakeRandomK8sName("config")
+	topicName := feature.MakeRandomK8sName("topic") // A k8s name is also a valid topic name.
+
+	f.Setup("create broker config", brokerconfigmap.Install(
+		configName,
+		brokerconfigmap.WithBootstrapServer(testpkg.BootstrapServersPlaintext),
+		brokerconfigmap.WithAuthSecret("does-not-exist"),
+	))
+
```
Comment (creydr) [inline test/rekt/features/broker_deleted_recreated.go:175]: Wouldn't it make sense to create the topic here, just to be sure, that that's not the reason the broker got deleted again?

Comment (matzew) [reply test/rekt/features/broker_deleted_recreated.go:175]:
```
// BrokerExternalTopicAuthSecretDoesNotExist tests that a broker can be deleted without the Secret or Topic existing.
```

**Review by creydr — APPROVED** [review]
/lgtm

**Pull request merged**